### PR TITLE
Unify logging in vnc/*

### PR DESCRIPTION
**Pull request by aquesnel**
Follow-on pull request to #1633 which migrates all logging in the vnc directory to use the LOG() and LOG_DEVEL() macros.

Changes:

* Migrate logging to the LOG() and LOG_DEVEL() macros (using text transform script)
* error logs use the LOG macro
* update code formatting with astyle for all files in xrdp/*

### CODE REVIEW

File: vnc/vnc.c [hunk 1]
```diff
@@ -1437,18 +1423,18 @@ lib_framebuffer_waiting_for_resize_confirm(struct vnc *v)
...
             else
             {
-                log_message(LOG_LEVEL_ERROR,
-                            "VNC server resize failed - error code %d [%s]",
-                            response_code,
-                            get_eds_status_msg(response_code));
+                LOG(LOG_LEVEL_WARNING,
+                    "VNC server resize failed - error code %d [%s]",
+                    response_code,
+                    get_eds_status_msg(response_code));
```
Comment (matt335672) [inline vnc/vnc.c:1434]: Agreed that WARNING is a better choice.

File: vnc/vnc.c [hunk 2]
```diff
@@ -2017,21 +2002,21 @@ lib_mod_connect(struct vnc *v)
             }
             else if (i == 0)
             {
-                log_message(LOG_LEVEL_DEBUG, "VNC Server will disconnect");
+                LOG(LOG_LEVEL_DEBUG, "VNC Server will disconnect");
```
Comment (matt335672) [inline vnc/vnc.c:2005]: LOG_LEVEL_ERROR?

File: vnc/vnc.c [hunk 3]
```diff
@@ -2017,21 +2002,21 @@ lib_mod_connect(struct vnc *v)
...
                 error = 1;
             }
             else
             {
-                log_message(LOG_LEVEL_DEBUG, "VNC unsupported security level");
+                LOG(LOG_LEVEL_DEBUG, "VNC unsupported security level");
```
Comment (matt335672) [inline vnc/vnc.c:2010]: Please replace with:

```
LOG(LOG_LEVEL_ERROR, "VNC unsupported security level %d", i);
```

**Review by matt335672 — COMMENTED** [review]
Excellent work as ever. Two minor comments.

File: vnc/vnc.c (hunk 2 above)
Comment (aquesnel) [reply vnc/vnc.c:2005]: done

File: vnc/vnc.c (hunk 3 above)
Comment (aquesnel) [reply vnc/vnc.c:2010]: done

**Comment by aquesnel** [comment]
@matt335672 I've implemented the feedback, let me know if anything else needs to be changed before this can be merged.

**Pull request merged**